# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/netpbm-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- netpbm-10.25/lib/libpbm3.c.orig	2005-02-03 07:57:27.000000000 +0000
+++ netpbm-10.25/lib/libpbm3.c	2005-02-03 07:57:31.000000000 +0000
@@ -80,6 +80,7 @@
 
     di const zero64 = 0;        /* to clear with PXOR */
 
+    int col;
     for (col =0; col < cols-7; col += 8)
         packedBits[col/8] 
           = bitreverse [ ~ (unsigned char) __builtin_ia32_pmovmskb (
